WORKSPACE

```diff
@@ -22,108 +22,156 @@ workspace(name = "vaticle_typedb_examples")
 ################################
 
 load("//dependencies/vaticle:repositories.bzl", "vaticle_dependencies")
+
```

Comment:
Let's get rid of these newlines, seems like a lot of them were added in this file